System/Process/Internals.hs

```diff
@@ -414,6 +429,8 @@ foreign import ccall unsafe "runInteractiveProcess"
         -> Ptr FD
         -> Ptr FD
         -> Ptr FD
+        -> Ptr CGid
```

Comment:
I've made this change in the latest commit, but I'm worried that there could be conflicts since `uid_t` and `gid_t ` are unsigned. It might be better to go with the original.